postgresql/migrations/000010_notification_subscription.up.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE public.notification_subscriptions (
+    id bigserial primary key,
+	pubkey bytea NOT NULL,
+	url varchar NOT NULL,
+	created_at bigint NOT NULL,
+	updated_at bigint NOT NULL
```

Comment:
updated_at is used in the code as a refreshed_at or last_registered_at. Nothing is updated (at least with the current code). The field name is wrong imo. Beside that, do we need a way to really update (when a device id changes for instance)?

Reply:
@yaslama  If the sdk register the same url we want to have the last registration timestamp.

Reply:
@yaslama yeah it makes sense to have a way to change to a new url. In the current code that would end up being two registrations. I'd like to pull that logic out of this PR though, because even though it seems like a small thing, it's pretty complex and I want to work on that isolated.

I'll change the name to `refreshed_at`

Reply:
Perhaps a good way to handle update is  to add a field app_id and mage oubkey, app_id compound unique so every app I'd can update its own url?
I think we should keep the client side very simple.

Reply:
@roeierez yeah a client could manage its own subscription id. So (pubkey, subscription_id) would become unique and we'd update the url on every call, so it could potentially be changed at some point. The client/app would then have to make sure that it always sends the same id for its subscription.

Reply:
Ook but for now I am taking back my comment as I carried away and complicated something simple.
I am in favor of not doing anything regarding update and leave as is.
We can simply and easily remove stale urls later.
What do you think @yaslama @JssDWt ?